src/pyOpenMS/tests/unittests/test_chemistry.py

```
@@ -0,0 +1,653 @@
+  assert seq.getResidue(1) is not None
+  assert seq.size() == 16
+
+  # test exception forwarding from C++ to python
+  # classes derived from std::runtime_exception can be caught in python
+  try:
+    seq.getResidue(1000) # does not exist
+  except RuntimeError:
+    print("Exception successfully triggered.")
+  else:
+    print("Error: Exception not triggered.")
+    assert False
```

Ruff (0.14.10)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)